diff --git a/web/modules/contrib/br_address_field/src/Plugin/Field/FieldFormatter/BrAddressPlainFormatter.php b/web/modules/contrib/br_address_field/src/Plugin/Field/FieldFormatter/BrAddressPlainFormatter.php
index aed824b..303b2d8 100644
--- a/web/modules/contrib/br_address_field/src/Plugin/Field/FieldFormatter/BrAddressPlainFormatter.php
+++ b/web/modules/contrib/br_address_field/src/Plugin/Field/FieldFormatter/BrAddressPlainFormatter.php
@@ -60,7 +60,7 @@ class BrAddressPlainFormatter extends FormatterBase {
             1 => $this->t('Initials'),
         ];

-        $summary[] = $this->t('How to show state field: @style', ['@style' => $display_state_options[$this->getSetting('display_state')]]);
+        $summary[] = $this->t('How to show state field: @style', ['@style' => $display_state_options[$this->getSetting('display_state')] ?? ""]);
         return $summary;
     }

@@ -102,7 +102,7 @@ class BrAddressPlainFormatter extends FormatterBase {

         foreach ($items as $delta => $item) {
             $values = $item->getValue();
-
+            $state = "RS";
             switch ($this->getSetting('display_state')) {
                 case 0:
                     $state = $states[$values['state']];
